stories/example-screen-staff-list.stories.js

```diff
@@ -27,150 +17,1023 @@ export default {
+          <button type="button" class="btn btn-sm btn-danger">
+            <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+              <use xlink:href="static/media/node_modules/@symplicity/tabler-icons/symplicity/tabler-sprite-nostroke.svg#tabler-plus" />
+            </svg>Add New
+          </button>
+        </div>
+        <div class="bg-white my-3 w-100">
+          <div class="text-center">${tabsDoubleRowHTML}</div>          
+          <div class="p-4">
+            <div style="background-color: #F6F6F6;">
+              <form class="row p-3 m-0 align-items-center">
+                <div class="col-md-6 col-lg-3 my-1">
+                  <input type="text" class="form-control fs-6" id="text" placeholder="Search cases by student, email, IR#" aria-label="Search">
+                </div>
+                <div class="col-md-6 col-lg-2 my-1">
+                  <select class="form-select fs-6" aria-label="Default select example">
+                    <option selected>Assigned to</option>
+                    <option value="1">One</option>
+                    <option value="2">Two</option>
+                    <option value="3">Three</option>
+                  </select>
+                </div>
+                <div class="col-md-6 col-lg-3 d-flex flex-wrap my-1">
+                  <div class="form-check me-3">
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheck1">
+                    <label class="form-check-label fs-6" for="flexCheck1">
+                      Parent Case
+                    </label>
+                  </div>
+                  <div class="form-check me-3 my-1">
+                    <input class="form-check-input type="checkbox" value="" id="flexCheck2">
+                    <label class="form-check-label fs-6" for="flexCheck2">
+                      Child Case   
+                    </label>
+                  </div>
+                </div>
+                <div class="col-md-6 col-lg-4 d-flex justify-content-end flex-wrap my-1">
+                  <button type="button" class="btn btn-outline-secondary ms-2 fs-6">
+                    <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                      <use xlink:href="static/media/node_modules/@symplicity/tabler-icons/symplicity/tabler-sprite-nostroke.svg#tabler-search" />
+                    </svg>Apply Search
+                  </button>
+                  <button type="button" class="btn btn-outline-secondary ms-2 fs-6">
+                    <svg width="1.125rem" height="1.125rem" class="mt-n1 me-1">
+                      <use xlink:href="static/media/node_modules/@symplicity/tabler-icons/symplicity/tabler-sprite-nostroke.svg#tabler-adjustments-horizontal" />
+                    </svg>More Filters
+                  </button>
+                </div>
+              </form>
+            </div>
+            <div class="d-flex align-items-center justify-content-between flex-wrap py-3">
```

Comment:
Lines 989-1023 should probably be broken out into `${batchAndPagination}`. There are also a number of visual inconsistencies here that we need to address:

![2021-07-09_10-09-04AM](https://user-images.githubusercontent.com/1605754/125099815-0b103c00-e09e-11eb-8048-401b3a246b4f.png)